tests/unit/test_emr.py

```diff
@@ -64,6 +64,8 @@ def test_cluster(bucket, cloudformation_outputs, emr_security_configuration):
     step_state = wr.emr.get_step_state(cluster_id=cluster_id, step_id=step_id)
     assert step_state == "PENDING"
     wr.emr.terminate_cluster(cluster_id=cluster_id)
+    while "TERMINATED" not in wr.emr.get_cluster_state(cluster_id=cluster_id):
```

Comment:
My only worry here is if "TERMINATED" state is never reached then it would be an infinite loop

Would an auto-termination policy do the trick instead?
https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-auto-termination-policy.html

Reply:
Great point.. let me see how quickly the auto-termination policy takes down the cluster in practice.

Reply:
@jaidisido I still need to check for terminated status before test teardown so I added a timeout to avoid the above.

Let me know your thoughts.